packages/pn-personafisica-webapp/src/api/notifications/__test__/Notifications.api.test.ts

```diff
@@ -177,25 +220,37 @@ describe('Notifications api tests', () => {
     expect(res).toStrictEqual({
       checkoutUrl: 'mocked-url',
     });
-    mock.reset();
-    mock.restore();
+    cleanupMock(mock);
   });
 
   it('exchangeNotificationQrCode', async () => {
-    const mock = new MockAdapter(apiClient);
-    mock.onPost(NOTIFICATION_ID_FROM_QRCODE(), { aarQrCodeValue: 'qr1' }).reply(200, {
-      iun: 'mock-notification-1',
-      mandateId: 'mock-mandate-1',
-    });
+    const mock = mockApi(
+      apiClient,
+      'POST',
+      NOTIFICATION_ID_FROM_QRCODE(),
+      200,
+      { aarQrCodeValue: 'qr1' },
+      {
+        iun: 'mock-notification-1',
+        mandateId: 'mock-mandate-1',
+      }
+    );
     mock.onPost(NOTIFICATION_ID_FROM_QRCODE(), { aarQrCodeValue: 'qr2' }).reply(200, {
```

Comment:
the mock of qr2 is useless. you can remove it

Reply:
changes made into 1bf7836db3775121009b8ac49703ef1bd0689074